### PR TITLE
Added saved and popular workouts to homeScreen

### PR DESCRIPTION
**Pull request by emilsavino**
We need to change a few things workoutView to make everything work as it should. Plus we still need a way to fetch the saved and popular workouts, but it should be easy to adjust once we have that

### CODE REVIEW

File: src/main/java/Group15/Model/Workout.java [hunk 1]
```diff
@@ -6,11 +6,29 @@
...
 {
     private final List<Exercise> exercises = new ArrayList<>();
+    private String name;
+    private String description;
 
     public Workout()
     {
     }
 
+    public void setName(String name) {
```
Comment (WettyWednesday) [inline src/main/java/Group15/Model/Workout.java:16]: Maybe give the name bold text? So they dont blend in together

File: src/main/java/Group15/View/HomeScreenView.java [hunk 2]
```diff
@@ -44,21 +49,16 @@ private static Pane createTitlePane()
...
-        quickWorkoutPane.setBackground(new Background(new BackgroundFill(Color.YELLOWGREEN, null, null)));
-        Label quickWorkoutLabel = new Label("A list of some sort of workouts could be displayed " +
-                "here in the colored area. They could have a title and a description\nand be formatted in a way" +
-                "where we can have 5-6 workouts displayed.");
-        Label additionalComment = new Label("The color should not be green btw, it's just to show the area to use");
-        quickWorkoutPane.getChildren().add(quickWorkoutLabel);
-        quickWorkoutPane.getChildren().add(additionalComment);
+        Pane savedWorkoutsPane = createWorkoutsPane("Saved");
+        Pane popularWorkoutsPane = createWorkoutsPane("Popular");
+        quickWorkoutPane.getChildren().addAll(savedWorkoutsPane, popularWorkoutsPane);
```
Comment (WettyWednesday) [inline src/main/java/Group15/View/HomeScreenView.java:61]: I cant comment further down, but the "New Workout" button should open the MuscleView instead. The "My Workout" button can open the WorkoutView

**Review by WettyWednesday — CHANGES_REQUESTED** [review]

File: src/main/java/Group15/View/HomeScreenView.java (hunk 2 above)
Comment (emilsavino) [reply src/main/java/Group15/View/HomeScreenView.java:61]: i know but someone haven't merged that one yet :)

Comment (WettyWednesday) [reply src/main/java/Group15/View/HomeScreenView.java:61]: Check again ;)

**Review by WettyWednesday — APPROVED** [review]

**Review by FredBas — APPROVED** [review]

**Pull request merged**